arnoldPlugins/gaffer.mtd

```diff
@@ -199,79 +203,102 @@
 
 	gaffer.layout.section.Shape.collapsed BOOL false
 	gaffer.layout.section.Sampling.collapsed BOOL false
+	gaffer.layout.activator.modeIsPortal STRING "parameters['portal'].getValue() == False"
 
 	[attr portal]
 		options STRING "Light:0|Portal:1"
 		label STRING "Mode"
 		gaffer.layout.index INT 0
 	[attr color]
 		gaffer.layout.index INT 1
+		gaffer.layout.visibilityActivator STRING "modeIsPortal"
 	[attr intensity]
 		gaffer.layout.index INT 2
+		gaffer.layout.visibilityActivator STRING "modeIsPortal"
 	[attr exposure]
 		gaffer.layout.index INT 3
+		gaffer.layout.visibilityActivator STRING "modeIsPortal"
 	[attr normalize]
 		gaffer.layout.index INT 4
+		gaffer.layout.visibilityActivator STRING "modeIsPortal"
 	[attr aov]
 		gaffer.layout.index INT 5
+		gaffer.layout.visibilityActivator STRING "modeIsPortal"
 
 	[attr roundness]
 		page STRING "Shape"
 		gaffer.layout.index INT 6
+		gaffer.layout.visibilityActivator STRING "modeIsPortal"
 	[attr soft_edge]
 		page STRING "Shape"
 		gaffer.layout.index INT 7
+		gaffer.layout.visibilityActivator STRING "modeIsPortal"
 	[attr spread]
 		page STRING "Shape"
 		gaffer.layout.index INT 8
+		gaffer.layout.visibilityActivator STRING "modeIsPortal"
 	[attr resolution]
 		page STRING "Shape"
 		gaffer.layout.index INT 9
```

Comment:
Should we also be applying the activator to `resolution`?

Reply:
I had thought so too, but at least in the latest 3ds Max release, it's enabled when using portal mode. I'll do a test to see if I can find an actual difference in the rendered result.

Reply:
The docs talk about it being the resolution of the importance table used to sample the input texture, so I can't imagine it being useful in portal mode. I could always be wrong though...

Reply:
After testing, the resolution on the portal has not effect. Fixed in a8e4654d35b1737af40b493f27e77f889781bafa